src/etapi/validators.ts

```diff
@@ -24,15 +24,15 @@ function isString(obj: ValidatorArg) {
 }
 
 function isLocalDateTime(obj: ValidatorArg) {
```

Comment:
I missed this in the previous review:
These functions will never work, since the type of the parameter 'ValidatorArg' is `object | undefined | null`. So 'obj' can never be of type string.

Reply:
@alexpietsch , good point.
I've added `string` to `ValidatorArg` as well, so it should be OK now.

Reply:
@eliandoran In general, do you think it makes sense to have a generic type for this? Because essentially with this you allow to pass arguments to this function, even if they are the wrong type. But on the other side I also see the benefit since it's simpler.

Reply:
@alexpietsch , I think it would actually make more sense to use `unknown` instead of `ValidatorArg`. The reason is that these validators are used strictly for user input provided via the ETAPI.

I've made the change, let me know if it works for you.